includes/find3ai.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Framework for Internal Navigation and Discovery AI
+Documentation=https://www.internalpositioning.com/doc/
+After=mosquitto.service
+
+[Service]
+WorkingDirectory=/opt/find3/server/ai
+RuntimeDirectory=find3ai
```

Comment:
I think it should be the same rundir as find3

Reply:
ok